permits/serializers.py

```diff
@@ -16,20 +21,116 @@ def to_representation(self, value):
         return list(meta_types)
 
 
-class PermitRequestSerializer(serializers.ModelSerializer):
+class WorksObjectTypesNames(serializers.RelatedField):
+    def to_representation(self, value):
+        wot_names = {wot.id: str(wot) for wot in value.all()}
+        return wot_names
+
 
+class PermitRequestSerializer(serializers.ModelSerializer):
     administrative_entity = PermitAdministrativeEntitySerializer(
         many=False, read_only=True
     )
     meta_types = MetaTypesField(source="works_object_types", read_only=True)
+    works_object_types_names = WorksObjectTypesNames(
+        source="works_object_types", read_only=True
+    )
 
     class Meta:
         model = models.PermitRequest
-        fields = ("status", "administrative_entity", "works_object_types", "meta_types")
+        fields = (
+            "id",
+            "status",
+            "administrative_entity",
+            "works_object_types",
+            "creditor_type",
+            "intersected_geometries",
+            "meta_types",
+            "works_object_types_names",
+        )
 
 
-class PermitRequestGeoTimeSerializer(gis_serializers.GeoFeatureModelSerializer):
+class PermitRequestAmendPropertyValueSerializer(serializers.RelatedField):
+    def to_representation(self, value):
+        works_object_types = [
```

Comment:
Je suis pas sûr de comprendre l’utilité de ce filtre, vu qu’on récupère de toute façon uniquement les `PermitRequestAmendPropertyValue` qui sont liées à cette demande de permis?

Reply:
Prefetched and refactored